src/app/app.module.ts

```diff
@@ -45,6 +46,7 @@ import { ProfileComponent } from './components/profile/profile.component';
     MatInputModule,
     MatButtonModule,
     MatSelectModule,
+    MatGridListModule
```

Comment:
missing comma here

Reply:
remove the <<<<< HEAD and =====, >>>>> merge information lines

Reply:
changes made and pushed. Thank you!